libs/blocks/global-navigation/global-navigation.js

```diff
@@ -63,25 +63,26 @@ const decorateSignIn = async ({ rawElem, decoratedElem }) => {
   let signInElem;
 
   if (!dropdownElem) {
-    signInElem = toFragment`<a href="#" daa-ll="${signInLabel}" class="feds-signIn">${signInLabel}</a>`;
+    signInElem = toFragment`<button qa="gnav-ims-sign-in" daa-ll="${signInLabel}" class="feds-signIn">${signInLabel}</button>`;
 
     signInElem.addEventListener('click', (e) => {
       e.preventDefault();
       signIn();
     });
   } else {
-    signInElem = toFragment`<a href="#" daa-ll="${signInLabel}" class="feds-signIn" role="button" aria-expanded="false" aria-haspopup="true">${signInLabel}</a>`;
+    signInElem = toFragment`<button qa="gnav-ims-sign-in" daa-ll="${signInLabel}" class="feds-signIn" aria-expanded="false" aria-haspopup="true">${signInLabel}</button>`;
```

Comment:
I have to say I'm not in love with this attribute. Attribute name is pretty generic, its value is prefixed with _gnav_, whereas all our prefixes are _feds_, it can represent either a sign in link OR both a dropdown trigger and a sign in link and its only purpose is to make unit tests a bit more convenient. I'm not convinced this is the right approach.

Since we decide what gets turned into a button and what remains a link, wouldn't just checking for the button be enough?

Reply:
Adding the `signIn` class to both now to share some styles. But you are right we could also just use the button to see which is the ims sign in!

Reply:
Too bad this attribute didn't make it as it would have given SDETs a way to target the button via attribute, instead of class, which isn't as reliable cause classes are not unique. Also, it would have started a best practice to add anchors for automation.